omero/sysadmins/unix/walkthrough/walkthrough_centos7.sh

```diff
@@ -98,6 +98,9 @@ psql -h localhost -U "$OMERO_DB_USER" "$OMERO_DB_NAME" < $OMERODIR/db.sql
 omero certificates
 #end-seclevel
 
+#start-diffie-hellman
+omero config set omero.glacier2.IceSSL.Ciphers=HIGH:!DH
```

Comment:
For the command to be kept in the doc, a  change will be required in omero-install.  @dominikl could you take care of it?

Reply:
PR open https://github.com/ome/omero-install/pull/272

Reply:
Changes upstream. https://github.com/ome/omero-documentation/pull/2332